rtil/src/native/livesplit.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::Mutex;
+use livesplit_core::{Segment, TimeSpan, Timer, TimingMethod};
+use once_cell::sync::Lazy;
+
+static LIVESPLIT_STATE: Lazy<Mutex<LiveSplit>> = Lazy::new(|| Mutex::new(LiveSplit::new()));
+
+pub struct LiveSplit {
+    pub run: livesplit_core::Run,
+    pub timer: Timer,
+}
+pub struct _Timer {}
+
+impl LiveSplit {
+    pub fn new() -> LiveSplit {
+        let mut run = livesplit_core::Run::new();
+        run.clone().set_game_name("Refunct");
+        run.clone().set_category_name("Any%");
```

Comment:
These clones are wrong and break what you actually want to do. The run is cloned, i.e., a new instance is created. Then you call the setter on the new instance and immediately throw it away.